gsa/src/gmp/models/cve.js

```diff
@@ -31,6 +31,16 @@ import Info from './info';
 class Cve extends Info {
   static entityType = 'cve';
 
+  static fromResultElement(element) {
```

Comment:
Hacky solution. Didn't want to touch CVEs. `Cve.fromElement` does not give the desired result.